node/devnet/src/lib.rs

```diff
@@ -56,6 +57,9 @@ pub fn start_from_client(client: Arc<Client>, devnet_cfg: DevNetConfig, rpc_cfg:
 
         // Spawn consensus tasks.
         spawn_consensus(client.clone(), consensus_tx, control_rx, devnet_cfg.block_period);
+
+        // Spawn tasks to consume not used channels.
+        tokio::spawn(out_block_rx.for_each(move |_| { future::ok(()) }));
```

Comment:
Why is it needed?

Reply:
Otherwise we are writing into channel that doesn't exist and get a lot of errors.
Errors should only appear in error situations.